src/ui/pages/common/PageTreeSelectorContainer.js

```diff
@@ -1,12 +1,14 @@
 
 import { connect } from 'react-redux';
 
-import PageTreeSelector from 'ui/pages/common/PageTreeSelector';
+import { PageTreeSelector } from '@entando/pagetreeselector';
 
 import { handleExpandPage } from 'state/pages/actions';
 
 import { getPageTreePages } from 'state/pages/selectors';
 
+import '@entando/pagetreeselector/dist/css/index.css';
```

Comment:
Current practice in appbuilder is importing css in appbuilder's index.js (see stylesheets import of @entando/menu and @entando/pages for reference).

@nicpuddu what do you think?

Reply:
np, I will move it there